client_python/README.md

```diff
@@ -1,433 +1,116 @@
----
-title: Grakn Python Client
-keywords: setup, getting started, download, driver
-tags: [getting-started]
-sidebar: documentation_sidebar
-permalink: /docs/language-drivers/client-python
-folder: docs
-symlink: true
----
+# Grakn Client Python
```

Comment:
Not sure about @marco-scoppetta  but I think it's more readable as `Grakn Python Client`

Reply:
Yeah it reads slightly better with the language name first. Grakn Java Client - Grakn Python Client - Grakn Node.js Client